### ISSUE TITLE
Update Input Labels When Adding Repeatable Rows

### ISSUE THREAD

**Issue by robincornett** [easydigitaldownloads/easy-digital-downloads#7836]
### Explain your enhancement (please be detailed)
When a row is added (for example, in the download metabox: Price ID, Download Files, or Software Licensing Upgrade Paths), labels associated with input fields should be updated to reference the new row input fields.

### Justification or use case
Currently, new row input field `name` and `id` are updated (if set), but the label `for` attribute is not. Many inputs do not have associated labels (#7701), but for those which do, updating the label attributes will improve both semantic markup and accessibility.

**Comment by robincornett** [comment]
Reopening as a JS error is thrown if a repeatable row has a label without a `for` attribute (see the Default setting for the Price ID in the Download Prices metabox).
